@navikt/core/react/src/pagination/pagination.stories.tsx

```diff
@@ -101,10 +101,30 @@ export const AsLink = () => {
     />
   );
 };
+
 AsLink.decorators = [
   (Story) => (
     <Router>
       <Story />
     </Router>
   ),
 ];
+
+export const Heading = () => {
+  const [page, setPage] = useState(1);
+  const props = {
+    page: 1,
+    count: 8,
+    siblingCount: 1,
+    boundaryCount: 1,
+  };
```

Comment:
Why have some of the props in an object?

Reply:
Think the change was added before the storybook were able to properly handle props and types. I'll fix 👍 

<img width="390" alt="Screenshot 2024-06-05 at 12 54 00" src="https://github.com/navikt/aksel/assets/26967723/900ff1ab-dc19-4cd3-a4e2-fb30cac1c1eb">